ring_api/server/rest-api.json

```diff
@@ -69,19 +69,14 @@
         "GET": "getSupportedCiphers"
     },
 
-    "/accounts/<account_id>/codecs/": {
-        "GET": {
-            "type": {
-                "active": "getActiveCodecList",
-                "details": "getCodecDetails"
-            }
-        },
-        "PUT": {
-            "type": {
-                "active": "setActiveCodecList",
-                "details": "setCodecDetails"
-            }
-        }
+    "/accounts/<account_id>/codecs/<codec_id>": {
+        "GET": "getCodecDetails",
+        "PUT": "setCodecDetails"
+    },
+
+    "/accounts/<account_id>/codecs/active/": {
```

Comment:
@Bl4ckb0ne `/accounts/<account_id>/codecs/` plus don't forget the */* at the end ! :#